app-emulation/virtualbox/virtualbox-7.0.4-r1.ebuild

```
@@ -336,6 +342,7 @@ src_configure() {
 		$(usex pulseaudio '' --disable-pulse)
 		$(usex python '' --disable-python)
 		$(usex vboxwebsrv --enable-webservice '')
+		$(usex vde --enable-vde '')
```

Note that with EAPI 8, you can do:
```suggestion
		$(usev vde --enable-vde)
```
(and `$(usev !foo --disable-foo)`)

---

Nice!